main.js

```diff
@@ -104,22 +107,42 @@ async function main() {
     const $timeoutInput = $('#timeout-input');
     $timeoutRange.val(detectorParams.timeout);
     $timeoutInput.val(detectorParams.timeout);
+    if (Cookies.get('timeoutInputSet') === 'true' && Cookies.get('timeoutInputValue')){
```

Comment:
Maybe we can change the line 77 to something like `timeout: Number(Cookies.get('timeout')) || 300,`

Here would be like `$timeoutInput.val(detectorParams.timeout)` and `$timeoutRange.val(detectorParams.timeout)`. Better yet since L108 is already covered that. We don't need any lines

Basically we shouldn't re update the right after the initialization since we have the ability to change the initial params.